.mise.toml

```diff
@@ -48,27 +48,74 @@ mv coverage.xml $root/
 
 [tasks."rust:lint"]
 description = "lint rust cargos"
+depends = ["platform:newsfeed:lint", "platform:discord:lint"]
+
+
+[tasks."rust:format"]
+description = "format rust cargos"
+depends = ["platform:newsfeed:format"]
+
+
+[tasks."rust:test"]
+description = "format rust cargos"
+depends = ["platform:newsfeed:test"]
+
+[tasks."rust:audit"]
+description = "audit rust cargos"
+depends = ["platform:newsfeed:audit"]
```

Comment:
I assume all of these general `rust` ones should be invoking all of the specific Rust Mise commands?

Reply:
yea, I think a lot of this is fixed upsteram